db/install.py

```diff
@@ -7,18 +8,25 @@
 def create_mathesar_database(
         user_database, username, password, hostname, root_database, port,
 ):
-    root_db_engine = engine.create_future_engine(
-        username, password, hostname, root_database, port,
-    )
-    with root_db_engine.connect() as conn:
-        conn.execution_options(isolation_level="AUTOCOMMIT")
-        conn.execute(text(f"CREATE DATABASE {user_database}"))
+    """Create database and install Mathesar on it."""
     user_db_engine = engine.create_future_engine(
         username, password, hostname, user_database, port
     )
-    install.install_mathesar_on_database(user_db_engine)
-    user_db_engine.dispose()
-    root_db_engine.dispose()
+    try:
+        user_db_engine.connect()
+        user_db_engine.dispose()
+        print(f"Database {user_database} already exists! Skipping...")
+    except OperationalError:
```

Comment:
Does this fix https://github.com/centerofci/mathesar/issues/1481?

Reply:
Yes, actually.

Reply:
Okay please mark #1481 a draft until you've updated it, @silentninja.